code/__DEFINES/sound.dm

```diff
@@ -26,7 +26,7 @@
 //THIS SHOULD ALWAYS BE THE LOWEST ONE!
 //KEEP IT UPDATED
 
-#define CHANNEL_HIGHEST_AVAILABLE 1015
+#define CHANNEL_HIGHEST_AVAILABLE 1013
 
 #define MAX_INSTRUMENT_CHANNELS (128 * 6)
```

Comment:
can we put these up right below the thing they're meant to be in sync with? maybe then people won't desync so easy

Reply:
done